core/src/test/scala/integration/kafka/api/PlaintextConsumerCommitTest.scala

```diff
@@ -271,6 +272,19 @@ class PlaintextConsumerCommitTest extends AbstractConsumerTest {
     consumer.commitSync()
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  @Timeout(15)
+  def testPositionRespectsTimeout(quorum: String, groupProtocol: String): Unit = {
```

Comment:
This test covers a position API behaviour that is unrelated to the committed offsets as I see it (we expect exactly the same when retrieving positions based on the partitions offsets from the leader instead of the committed offsets from the coordinator), so I would say it shouldn't be in this file ? Since there is not specific file for positions-related tests I would suggest we move it to the generic `PlainTextConsumerTest`. Makes sense?

Reply:
> Since there is not specific file for positions-related tests I would suggest we move it to the generic `PlainTextConsumerTest`. Makes sense?

I agree. Before I stuck it there, I looked around for any other tests named `test*Position*`. The only test I saw was `testPositionAndCommit` in `PlaintextConsumerCommitTest`, so 🤷‍♂️

Reply:
Done.